### PR TITLE
[PB-2981]: feat/[B2B – Drive] - fuzzy search for b2b items

### PR DESCRIPTION
**Pull request by Jona-Internxt**
Searching for items in B2B doesn't return any result, instead it shows the items in B2C. When clicking on a result, the app crashes.

### CODE REVIEW

**Comment by Jona-Internxt** [comment]
For merge this, we need this to be merged from backend

https://github.com/internxt/drive-server-wip/pull/419

**Review by xabg2 — APPROVED** [review]

**Review by rafijv — APPROVED** [review]

File: src/drive/storage/index.ts [hunk 1]
```diff
@@ -584,13 +584,13 @@ export class Storage {
...
    * @param {string} search - The name of the item.
+   * @param {string} workspaceId - The ID of the workspace (optional).
    * @returns {[Promise<SearchResultData>, RequestCanceler]} An array containing a promise to get the API response and a function to cancel the request.
    */
-  public getGlobalSearchItems(search: string): [Promise<SearchResultData>, RequestCanceler] {
-    const { promise, requestCanceler } = this.client.getCancellable<SearchResultData>(
-      `fuzzy/${search}`,
-      this.headers(),
-    );
+  public getGlobalSearchItems(search: string, workspaceId?: string): [Promise<SearchResultData>, RequestCanceler] {
```
Comment (xabg2) [inline src/drive/storage/index.ts:590]: Revert this change and add the endpoint to workspaces/index.ts using that client.

**Review by xabg2 — CHANGES_REQUESTED** [review]

**Review by xabg2 — APPROVED** [review]
